src/pool/srv_pool.c

```diff
@@ -2562,6 +2562,176 @@ ds_pool_svc_set_prop(uuid_t pool_uuid, d_rank_list_t *ranks, daos_prop_t *prop)
 	return rc;
 }
 
+/*
```

Comment:
nit: "/**" would be consistent with other function comments

Reply:
In this case since it's a non-public function, I don't think we want the comment showing up in the public docs generated by Doxygen.